_data/donation-human-needs.yml

```diff
@@ -44,10 +44,9 @@ websites:
   twitter: FareShare1
   region: oc
   country: AU
-  bch: true
-  btc: true
-  othercrypto: true
-  doc: https://twitter.com/FareShare1/status/949041934453981185
+  bch: false
+  btc: false
+  othercrypto: false
```

Comment:
Hmm, they still have 5 BCH in their donation address:

https://blockchair.com/bitcoin-cash/address/3JLehRBNg1oH4tyCZ4Jy1h1e3DwZzU5Nqm